downstream/modules/eda/proc-eda-verify-event-streams-work.adoc

```diff
@@ -0,0 +1,24 @@
+[id="eda-verify-event-streams"] 
+
+= Verifying your event streams work
+
+Verify that you can use your event stream to connect to a remote system and receive data.
+
+. Log in to {PlatformNameShort}.
+. From the navigation panel, select {MenuADEventStreams}.
+. Select the event stream that you created to validate connectivity and ensure that the event stream sends data to the rulebook activation. 
+. Verify that the events were received.You can see in the *Events received* field that the event was received. You can also see the header for the event stream that contains details about the event.
++
+image:eda-verify-event-streams.png[Verify event streams work]
++
+If you scroll down in the UI, you can also see the body of the payload with more information about the webhook. 
++
+image:eda-payload-body-event-streams.png[Payload body]
++
+
+The *Header* and *Body* sections for the event stream are displayed on the Details page. They differ based on the vendor who is sending the event. The header and body can be used to check the attributes in the event payload,  which will help you in writing conditions in your rulebook. For example:
+
+. Toggle the *Forward events to rulebook activation* option to  enable you to push your events to a rulebook activation.
+This moves the event stream to production mode and makes it easy to attach to rulebook activations.
++
+When this option is toggled off, your ability to forward events to a rulebook activation is disabled and you will see the *This event stream is disabled* message.
```

Comment:
and the * This event stream is disabled * message is displayed.

Reply:
Done.